Documentation/Contributing/documentation.md

```diff
@@ -6,12 +6,33 @@ We are using [MkDocs](https://www.mkdocs.org/) with the [Material for MkDocs the
 
 ## Markdown Extensions
 
-Thanks to the MkDocs theme we have certain "markdown syntax extensions" available:
+Thanks to the MkDocs Material theme we have certain "markdown syntax extensions" available:
 
-* Admonitions
-* Footnotes
-* Icons, Emojis
-* Lists (Task lists)
-* And some more
+* [Admonitions (also known as *call-outs*)](https://squidfunk.github.io/mkdocs-material/reference/admonitions/)
+* [Footnotes](https://squidfunk.github.io/mkdocs-material/reference/footnotes/)
+* [Icons, Emojis](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/)
+* Lists ([Task lists](https://squidfunk.github.io/mkdocs-material/reference/lists/#using-definition-lists))
+* And [more](https://squidfunk.github.io/mkdocs-material/reference/)..
 
 For a whole list of features [Reference - Material for MkDocs](https://squidfunk.github.io/mkdocs-material/reference/).
+
+## Local Preview and Build
+
+To locally preview or build the current state of the documentation (in the root of the repository):
+
+```console
+make docs-preview
+# or to build the docs
+make docs-build
```

Comment:
All we generally need during development is `make docs-prview`, right? How about deleting this line? Otherwise they wonder if they also need to build even after previewing. Or is building also useful to see if there are broken links?
```suggestion
```